### PR TITLE
Leaderboard: final touches

### PR DESCRIPTION
**Pull request by billymohajeri**
In this step I,
-  Improved the look and feel of the application, adding some styles.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,11 @@
 
 **Leaderboard** The leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service.
 
+<div align="center">
+
+  <img src="./app-screenshot.png" alt="app-screenshot" width="auto"  height="auto" />
+  </div>
+
```
Comment (GzimAsani) [inline README.md:40]: - [ ] The live version is different, kindly update it 
image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello  @billymohajeri                                :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Your styling looks amazing, good work+1: 

 ## Required Changes 

- [ ] Check the comments below :arrow_double_down: 



 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (billymohajeri) [reply README.md:40]: Thanks @GzimAsani,
it's done now.

**Review by levy002 — APPROVED** [review]
## Hello @billymohajeri 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] Kindly make sure to check again the link you deployed because it is broken.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**